data/sources/supporting-zoning.json

```diff
@@ -7,6 +7,16 @@
       "sql": "SELECT cartodb_id AS id, * FROM dcp_commercial_overlays",
       "dataPipeline": true
     },
+    {
+      "id": "zoning-map-index",
+      "sql": "SELECT cartodb_id AS id, CASE WHEN sectionalm ILIKE '%Subplan' THEN sectionalm ELSE dzm_sect_n END AS zmi_label , * FROM dcp_zoning_map_index",
+      "dataPipeline": true
+    },
+    {
+      "id": "zoning-map-index-centroids",
+      "sql": "SELECT ST_Centroid(the_geom_webmercator) as the_geom_webmercator, CASE WHEN sectionalm ILIKE '%Subplan' THEN sectionalm ELSE dzm_sect_n END AS zmi_label FROM dcp_zoning_map_index",
```

Comment:
Just need to do some more SQL string magic to get these to show up as `30d Sub plan` instead of `30d-subplan`. Applies here and to the tooltips.